db/schema.rb

```diff
@@ -0,0 +1,48 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2021_05_14_123355) do
+
+  create_table "event_attendee", force: :cascade do |t|
+    t.integer "attendee_id", null: false
+    t.integer "attended_event_id", null: false
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["attended_event_id"], name: "index_event_attendee_on_attended_event_id"
+    t.index ["attendee_id"], name: "index_event_attendee_on_attendee_id"
+  end
+
+  create_table "events", force: :cascade do |t|
+    t.date "date"
+    t.string "title"
+    t.string "Location"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.integer "creator_id"
```

Comment:
an index for creator_id column should be added.

Why indices for foreign keys are necessary? In short:
Since the foreign key columns will be used often to retrieve related information, it's important to index them (queries on those columns will be faster with indexes). More details

- [ ] [See this](https://semaphoreci.com/blog/2017/05/09/faster-rails-is-your-database-properly-indexed.html)